sktime/libs/momentfm/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""Python implementations of momentfm forecaster.
+
+Unofficial fork of the ``momentfm`` package, maintained in ``sktime``.
```

Comment:
we should make clear this is temporary until, hopefully, a proper package emerges.